charts/tekton-pipelines/templates/common/tasks/sentry-release.yaml

```diff
@@ -0,0 +1,82 @@
+{{ if .Values.sentry.enabled }}
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: sentry-release
+  namespace: {{ .Release.Namespace }}
+
+spec:
+  description: >-
+    The task prepares and deploys sentry release for projects, which exist in Sentry.
+
+  workspaces:
+    - name: source
+
+  resources:
+    inputs:
+      - name: app
+        type: git
+
+  params:
+    - name: project
+      type: string
+      description: name of the project, which component is deployed
+
+    - name: component
+      type: string
+      description: name of the application component, which is deployed
+
+    - name: environment
+      type: string
+      description: environment name of the app being built, i.e. dev/staging/prod
+
+    - name: sourcemaps_dir
+      type: string
+      description: name of the dir where frontend sourcemaps would be stored in workspace
+      default: "sourcemaps"
+
+  stepTemplate:
+    env:
+      - name: SENTRY_AUTH_TOKEN
+        valueFrom:
+          secretKeyRef:
+            name: {{ .Values.sentry.authTokenSecret }}
+            key: {{ .Values.sentry.authTokenSecretKey }}
+      - name: SENTRY_ORG
+        value: {{ .Values.sentry.org }}
+      - name: SENTRY_URL
+        value: {{ .Values.sentry.url }}
+
+  steps:
+    - name: release
+      image: {{ .Values.images.sentry_cli }}
+      imagePullPolicy: {{ .Values.imagePullPolicy }}
+      workingDir: $(resources.inputs.app.path)
+      script: |
+        #!/usr/bin/env sh
+        set +x
+
+        PROJECT_NAME="$(params.project)-$(params.component)"
```

Comment:
I think we should not hardcode the "project sentry name" this way and instead pass it as a trigger binding in the tekton-apps
https://github.com/saritasa-nest/saritasa-devops-helm-charts/blob/main/charts/tekton-apps/templates/general/triggerbindings.yaml

Reply:
that way if our naming change - we don't need to modify this chart.

Reply:
Maybe, but I am not sure that we will change projects names for Sentry projects. Basically Sentry projects names are `repos` names (that is how it is implemented in terraform - https://github.com/saritasa-nest/allstar-elevator-infra-dev-aws/blob/main/project-backend-dev.tf#L73).

So I think that we can add this feature once we need it, but I don't think that we will need it. What do you think?

Reply:
what I'm trying to say - you code right now has a knowledge HOW TO NAME the project based on project/component combination. That is coupling which will increase the maintenance cost down the road should we decide to change naming the naming convention.

It's not a big deal now, but may fire back in the future.

Reply:
done